### PR TITLE
[efl-util][ACR-1266]

### PR DESCRIPTION
**Pull request by jhyuni**
Add guides for efl_util_input_initialize_generator_with_sync().

### API Changes ###
 - ACR: ACR-1266

### CODE REVIEW

File: docs/application/native/guides/ui/efl/efl-util.md [hunk 1]
```diff
@@ -320,6 +323,9 @@ To generate touch input events:
        /*
           Since Tizen 4.0, you can create an input device with a given name (in this example, "Local Device"):
           inputgen = efl_util_input_initialize_generator_with_name(EFL_UTIL_INPUT_DEVTYPE_TOUCHSCREEN, "Local_Device");
+
+          Since Tizen 5.0, you can create an input generator synchronously with a given name (device_name can be NULL):
```
Comment (Reni-90) [inline docs/application/native/guides/ui/efl/efl-util.md]: Suggestion:
From Tizen 5.0, you can create an input generator synchronously with a given name (device_name can be NULL):

Is it "input generator" or "input device"?

File: docs/application/native/guides/ui/efl/efl-util.md [hunk 2]
```diff
@@ -261,6 +261,9 @@ To generate key input events:
        /*
           Since Tizen 4.0, you can create an input generator with a given name (in this example, "Local Device"):
           inputgen = efl_util_input_initialize_generator_with_name(EFL_UTIL_INPUT_DEVTYPE_KEYBOARD, "Local_Device");
+
+          Since Tizen 5.0, you can create an input generator synchronously with a given name (device_name can be NULL):
```
Comment (Reni-90) [inline docs/application/native/guides/ui/efl/efl-util.md]: Suggestion:
From, Tizen 5.0, you can create an input generator synchronously with a given name (device_name can be NULL):
Is it "input generator" or "input device"?

File: docs/application/native/guides/ui/efl/efl-util.md [hunk 3]
```diff
@@ -259,8 +259,11 @@ To generate key input events:
        /* Create an input device with the default name: "Input Generator" */
        inputgen = efl_util_input_initialize_generator(EFL_UTIL_INPUT_DEVTYPE_KEYBOARD);
        /*
-          Since Tizen 4.0, you can create an input generator with a given name (in this example, "Local Device"):
+          Since Tizen 4.0, you can create an input device with a given name (in this example, "Local Device"):
           inputgen = efl_util_input_initialize_generator_with_name(EFL_UTIL_INPUT_DEVTYPE_KEYBOARD, "Local_Device");
+
+          Since Tizen 5.0, you can create an input device synchronously with a given name (device_name can be NULL):
```
Comment (Reni-90) [inline docs/application/native/guides/ui/efl/efl-util.md]: Please update this sentence also as per the suggestions given in the comments.

Comment (jhyuni) [reply docs/application/native/guides/ui/efl/efl-util.md]: I don't understand what you said, please said more detail.
I alread fixed all sentense to input device not input generator.

Comment (Reni-90) [reply docs/application/native/guides/ui/efl/efl-util.md]: Hello,
Please consider the first part of the comment as well.

Suggestion:
From, Tizen 5.0, you can create an input generator synchronously with a given name (device_name can be NULL):

File: docs/application/native/guides/ui/efl/efl-util.md [hunk 4]
```diff
@@ -247,7 +247,7 @@ To take a screenshot:
 
 To generate key input events:
 
-1. Create the `efl_util_inputgen_h` structure and initialize the structure members with the `efl_util_input_initialize_generator()` or `efl_util_input_initialize_generator_with_name()` function:
+1. Create the `efl_util_inputgen_h` structure and initialize the structure members with the `efl_util_input_initialize_generator()` , `efl_util_input_initialize_generator_with_name()` , or `efl_util_input_initialize_generator_with_sync()` function:
```
Comment (mijin85cho) [inline docs/application/native/guides/ui/efl/efl-util.md]: Please remove one space before each comma. 

`efl_util_input_initialize_generator()`, `efl_util_input_initialize_generator_with_name()`, or `efl_util_input_initialize_generator_with_sync()` function

File: docs/application/native/guides/ui/efl/efl-util.md [hunk 5]
```diff
@@ -306,7 +309,7 @@ To generate key input events:
 
 To generate touch input events:
 
-1. Create the `efl_util_inputgen_h` structure and initialize the structure members with the `efl_util_input_initialize_generator()` or `efl_util_input_initialize_generator_with_name()` function:
+1. Create the `efl_util_inputgen_h` structure and initialize the structure members with the `efl_util_input_initialize_generator()` , `efl_util_input_initialize_generator_with_name()` , or `efl_util_input_initialize_generator_with_sync()` function:
```
Comment (mijin85cho) [inline docs/application/native/guides/ui/efl/efl-util.md]: Please remove one space before each comma. 

`efl_util_input_initialize_generator()`, `efl_util_input_initialize_generator_with_name()`, or `efl_util_input_initialize_generator_with_sync()` function

File: docs/application/native/guides/ui/efl/efl-util.md (hunk 4 above)
Comment (Reni-90) [reply docs/application/native/guides/ui/efl/efl-util.md]: I have asked @jhyuni to add one comma before "or".

Comment (mijin85cho) [reply docs/application/native/guides/ui/efl/efl-util.md]: Hello, @Reni-90 
And I'm asking @jhyuni to remove the **space** in front of comma, not comma itself.
We have additional spaces in front of each comma.

Comment (Reni-90) [reply docs/application/native/guides/ui/efl/efl-util.md]: I am sorry. 
I suggested the "comma issue". I think @jhyuni  misinterpreted. Sorry for that as well.

**Comment by jhyuni** [comment]
I removed space before comma, please check last commit

**Comment by mijin85cho** [comment]
@Reni-90, would you please check & approve the latest version again?

**Comment by Reni-90** [comment]
Sure, I will.

**Comment by mijin85cho** [comment]
Thanks for the review & update.

@jhyuni, I think from next version it would be better to remove line 260 and 322.
As we can't take all old version guide.

For now, I'll merge this PR and prepare the publish for Tizen 5.0 as it is now.

**Comment by mijin80lee** [comment]
TD Sync Completed.(for Tizen_5.0_prepare)

docs/application/native/guides/ui/efl/efl-util.md
line 192,193: Remove one space at the beginning.

**Pull request merged**